.github/workflows/publish-package-docs.yaml

```diff
@@ -60,11 +60,12 @@ jobs:
           python -m pip install setuptools six wheel
 
       - name: Install project requirements
-        run: |
-          pushd build-directory
-          python -m pip install -r requirements.txt
-          python -m pip install -r docs/docsite/requirements.txt
-          popd
+        run: python -m pip install -r requirements.txt
+        working-directory: build-directory
+
+      - name: Install docsite requirements
+        run: python -m pip install -r docs/docsite/requirements.txt
```

Comment:
Shouldn't this install `test/sanity/code-smell/docs-build.requirements.txt` instead, since these are the dependencies used by the "real" docsite build (except for the core devel build IIRC)?

Reply:
This is what I referred to in my other comments as the `if/then` statement needed in the future.  For devel or PRs - we use what's here. for /latest/ we'd use the code-smell one.

Reply:
That's different from what the real docsite uses, right? Because that one does not use the latest antsibull-docs version for docs.ansible.com/ansible/devel/, only for docs.ansible.com/ansible-core/devel/ (as you can see https://docs.ansible.com/ansible/devel/collections/ansible/builtin/url_lookup.html#parameter-ca_path vs. https://docs.ansible.com/ansible-core/devel/collections/ansible/builtin/url_lookup.html#parameter-ca_path - the env variable is only clickable for ansible-core devel, not for ansible devel).

Reply:
Ah good point. Too many jenkins jobs to remember what's what!  But yes, core has the open requirements for its nightly build and the rest use the code-smell version.